src/main.js

```diff
@@ -1,12 +1,39 @@
-import Vue from 'vue'
-import App from './App.vue'
-import router from './router'
-import store from './store'
+import Vue from 'vue';
+import App from './App.vue';
+import router from './router';
+import store from './store';
 
-Vue.config.productionTip = false
+Vue.config.productionTip = false;
+
+import * as VueGoogleMaps from 'vue2-google-maps';
+
+Vue.use(VueGoogleMaps, {
+	load: {
+		key: process.env.VUE_APP_GOOGLE_MAPS_API_KEY,
+		libraries: 'places', // This is required if you use the Autocomplete plugin
+		// OR: libraries: 'places,drawing'
```

Comment:
Could you delete these comments? We can always refer back to the documentation if we need to!

Reply:
I have deleted these comments and pushed up the newest changes. My apologies for missing those.

Reply:
No problem, thanks for making that update!